src/components/radioButtons.tsx

```diff
@@ -0,0 +1,25 @@
+import Form from "react-bootstrap/Form";
+interface Props {
+  levels: string[];
+}
+function RadioButtons(props: Props) {
+  return (
+    <Form>
+      <div key={`inline-radio`} className="mb-3">
+        {props.levels.map((l: string, index: number) => {
```

Comment:
```suggestion
        {props.levels.map((l: string, i: number) => {
```

Maybe follow the convention of one letter here? You already have `l`.